### DISCUSSION TITLE
动能发电机无法发电

### DISCUSSION THREAD

**Discussion by TelePortNX** [IndustrialCraftMC/IndustrialCraft2#94] [【漏洞反馈】Bugs Report]
我测试过程中 使用了风力发电(前面风力动能发生机 后面动能发电机) 我发现风力动能发生机显示输出32EU 但是动能发电机没有输出 我以为机制变更了 然后我把动能发电机打掉 把储电箱放到风力动能发生机后面 也没电力输出

**Comment by AbbottMc** [comment]
请检查是否按照正确的方向摆放了相关机械，以及尝试按照不同的顺序放置这两个机械，如果确认无误请提供相关摆放方式的截图
